SIL.Media/SIL.Media.csproj

```diff
@@ -34,7 +34,9 @@ See full changelog at https://github.com/sillsdev/libpalaso/blob/feature/nuget/C
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
-    <PackageReference Include="NAudio" Version="1.8.4" />
+    <PackageReference Include="SIL.NAudio" Version="1.9.0-beta0074">
+      <PrivateAssets>all</PrivateAssets>
```

Comment:
Do we really want `<PrivateAssets>all</PrivateAssets>` for `SIL.NAudio`? I'd think that we need the assembly at runtime, not just during the build.